src/doc/style-guide/src/README.md

```diff
@@ -32,6 +32,19 @@ This should not be interpreted as forbidding developers from following a
 non-default style, or forbidding tools from adding any particular configuration
 options.
 
+## Bugs
+
+If the style guide differs from rustfmt, that may represent a bug in rustfmt,
+or a bug in the style guide; either way, please report it to the style team or
+the rustfmt team or both, for investigation and fix.
+
+If implementing a new formatting tool based on the style guide and default Rust
+style, please test it on the corpus of existing Rust code, and avoid causing
+widespread breakage. The implementation and testing of such a tool may surface
```

Comment:
I mostly remember the meeting where we discussed this so I think I'm just confusing myself now. Is the implication that some new formatting tool implementor _could_ cause widespread breakage by testing their new tool on the corpus of existing Rust code, and we're asking them to perform that testing in a safe manner so that they _don't cause that breakage, or that if they don't test then their tool could cause widespread breakage?

In either case, I'm not entirely sure how such widespread breakage would occur, and I'm confused by what's being requested of me (if I'm wearing the hypothetical new tool author hat)

Reply:
@calebcartwright
> Is the implication that some new formatting tool implementor could cause widespread breakage by testing their new tool on the corpus of existing Rust code, and we're asking them to perform that testing in a safe manner so that they _don't cause that breakage, or that if they don't test then their tool could cause widespread breakage?

Yes, exactly. Implementing a new formatting tool based solely on the style guide, without testing it, may differ from rustfmt (either because rustfmt differs from the style guide or because the style guide missed defining some detail of the style), and would then cause breakage. We're asking people to test their tool against a large amount of existing Rust code and make sure they don't unexpectedly differ from rustfmt, and report bugs if they think they've implemented the default Rust style but end up with a result different from rustfmt.